client/components/mma/upgrade/UpgradeSupportSwitchThankYou.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+	Button,
+	LinkButton,
+	Stack,
+	SvgCalendar,
+	SvgClock,
+	SvgEnvelope,
+} from '@guardian/source-react-components';
+import { useContext } from 'react';
+import { useNavigate } from 'react-router';
+import {
+	sectionSpacing,
+	signInContentContainerCss,
+	signInCss,
+	signInHeadingCss,
+	signInParaCss,
+} from '../../shared/SignIn';
+import {
+	headingCss,
+	stackedButtonLayoutCss,
+	whatHappensNextCss,
+} from '../cancel/cancellationSaves/SaveStyles';
+import { Heading } from '../shared/Heading';
+import { SwitchSignInImage } from '../switch/complete/SwitchSignInImage';
+import { buttonCentredCss, iconListCss } from '../switch/SwitchStyles';
+import type { UpgradeSupportInterface } from './UpgradeSupportContainer';
+import { UpgradeSupportContext } from './UpgradeSupportContainer';
+
+export const UpgradeSupportSwitchThankYou = () => {
+	const upgradeSupportContext = useContext(
+		UpgradeSupportContext,
+	) as UpgradeSupportInterface;
+
+	const navigate = useNavigate();
+	const userEmailAddress = upgradeSupportContext.user;
+
+	return (
+		<>
+			<section css={sectionSpacing}>
+				<Stack space={4}>
+					<h2 css={headingCss}>
+						Thank you for supporting us with{' '}
+						contributionPriceDisplay/billingPeriod
+					</h2>
+				</Stack>
+			</section>
+			<section css={sectionSpacing}>
+				<Heading sansSerif>What happens next?</Heading>
+				<ul css={[iconListCss, whatHappensNextCss]}>
+					<li>
+						<SvgEnvelope size="medium" />
+						<span data-qm-masking="blocklist">
+							You will receive a confirmation email to{' '}
+							<> {userEmailAddress} </>
+						</span>
+					</li>
+					<li>
+						<SvgCalendar size="medium" />
+						<span>
+							Your first billing date is today and you will be
+							charged {upgradeSupportContext.mainPlan.currency}{' '}
+							amount payable today , your ongoing{' '}
+							{upgradeSupportContext.mainPlan.billingPeriod}{' '}
+							payment will be{' '}
+							{upgradeSupportContext.mainPlan.currency}next
+							payment amount
+						</span>
+					</li>
+					<li>
+						<SvgClock size="medium" />
+						<span>
+							Your new support will start today. It can take up to
+							an hour for your support to be activated
+						</span>
+					</li>
+				</ul>
+			</section>
+			<section>
+				<div css={signInCss}>
+					<SwitchSignInImage />
```

Comment:
I wonder if this banner would be worth making a component at this point - is it exactly the same as on the switch page?

Reply:
I have made a SignIn shared file and was starting to refactor